api/tests/unit/users/test_unit_users_views.py

```diff
@@ -608,32 +608,37 @@ def test_group_admin_can_retrieve_group(
     assert response.status_code == status.HTTP_200_OK
 
 
+def delete_user(
+    user: FFAdminUser,
+    password: str = None,
+    delete_orphan_organisations: bool = True,
+):
+    client = APIClient()
+    client.force_authenticate(user)
+    data = {
+        "delete_orphan_organisations": delete_orphan_organisations,
+    }
+    if password:
+        data["password"] = password
+
+    url = "/api/v1/auth/users/me/"
+    return client.delete(url, data=json.dumps(data), content_type="application/json")
+
+
 @pytest.mark.django_db
 def test_delete_user():
-    def delete_user(
-        user: FFAdminUser, password: str, delete_orphan_organisations: bool = True
-    ):
-        client = APIClient()
-        client.force_authenticate(user)
-        data = {
-            "current_password": password,
-            "delete_orphan_organisations": delete_orphan_organisations,
-        }
-        url = "/api/v1/auth/users/me/"
-        return client.delete(
-            url, data=json.dumps(data), content_type="application/json"
-        )
-
     # create a couple of users
     email1 = "test1@example.com"
     email2 = "test2@example.com"
     email3 = "test3@example.com"
+
```

Comment:
Could you please remove any unnecessary line breaks from the code?

Reply:
@shubham-padia are you able to resolve this and we can get this merged?